src/bluebot/tools/README.md

```
@@ -0,0 +1,254 @@
+2. **Configure your identity (optional):**
+
+   Set in `.env`:
+   ```bash
+   TEST_USER_ID=123456789012345678
+   TEST_USERNAME=YourName
+   TEST_NICKNAME=YourNick
+   BLUEBOT_ENEMY_USER_ID=999999999999999999  # For testing enemy behavior
+   ```
+
+3. **Start testing:**
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)